src/evaluation/Run.jl

```diff
@@ -146,7 +146,7 @@ will_run_code(notebook::Notebook) = notebook.process_status != ProcessStatus.no_
 ###
 
 "Do all the things!"
-function update_save_run!(session::ServerSession, notebook::Notebook, cells::Array{Cell,1}; save::Bool=true, run_async::Bool=false, prerender_text::Bool=false, kwargs...)
+function update_save_run!(session::ServerSession, notebook::Notebook, cells::Array{Cell,1}; save::Bool=true, prerender_text::Bool=false, kwargs...)
```

Comment:
Hey!! I may be missing something, but where is `run_async` now?

Reply:
> Hey!! I may be missing something, but where is `run_async` now?

It was part of the `kwargs` array, which are then passed to `maybe_run_async!`. 
I [put it back](https://github.com/fonsp/Pluto.jl/compare/f9bad8ff21d61afe146a4e368a636df7bf85b152..682d085d86648c827622b510881449364d7cc2d8) to be more explicit :smiley: